### ISSUE TITLE
Error System.AggregateException

### ISSUE THREAD

**Issue by Josegg0** [Ryochan7/DS4Windows#1991]
**Describe the bug**
03/24/2021 15:23:20: System.AggregateException: One or more errors occurred. ---> System.ArgumentException: An item with the same key has already been added. 

See full message in log.

**To Reproduce**
1. First Connect a DS4 via bluetooth.
2. Then Connect the same DS4 via USB cable while still connected via bt.
3. See error in log. (Status indicates bt and battery charging)
[BTthenwired.txt](https://github.com/Ryochan7/DS4Windows/files/6200332/BTthenwired.txt)

**Expected behavior**
Change DS4 connection from bluetooth to USB cable? (not sure about this) and/or a message in the log indicating the status.

**Screenshots**

**Desktop (please complete the following information):**
 - Controller Make and Model: tried with Sony DS4 v.1 and v.2 same results.
 - OS: Windows 10 home 20H2
 - DS4Windows Version 2.2.13-14 (don't know when this started)

**Additional context**

**Comment by RedDevilus** [comment]
![image](https://user-images.githubusercontent.com/24227051/112382945-c8904280-8cec-11eb-900d-c70514487af2.png)

Is that just the warning for this?
`Check for minimum .NET version on startup. Temporary measure warning about future upgrade` 

Seemed to go away when i connected DS4.

**Comment by TheMasterDingo** [comment]
Tested on mine, can confirm it

**Comment by Ryochan7** [comment]
For future reference.; The key factor here: **Shared mode**. There have been reports like this before that were not clear; a lot of times people did not include the Log file. Looking at the Log file and the DS4Devices class, I found a simple test and I was able to duplicate the problem. Almost all the time, I have the **Hide DS4 Controller** option checked so I would not run into this problem.

Due to a Microsoft problem mentioned in another issue, it looks like there will be a version 2.2.15 before version 3.0 is released. Expect a public fix later today.
